site/content/docs/5.3/helpers/ratio.md

```diff
@@ -22,11 +22,24 @@ Aspect ratios are declared in a Sass map and included in each class via CSS vari
 
 Wrap any embed, like an `<iframe>`, in a parent element with `.ratio` and an aspect ratio class. The immediate child element is automatically sized thanks to our universal selector `.ratio > *`.
 
-{{< example >}}
+<div class="bd-example">
+  <div class="ratio ratio-16x9 youtube_player" videoID="zpOULjyy-n8" relates="0" loading="0" allowfullscreen>
+    <div class="d-flex justify-content-center align-items-center bg-dark p-1 p-sm-3 p-xl-5">
+      <div class="alert alert-warning m-1 m-sm-3 m-xl-5">
+        <span class="alert-icon"><span class="visually-hidden">Warning</span></span>
```

Comment:
I haven't accepeted Youtube cookies:
![screenshot-deploy-preview-2089--boosted netlify app-2023 06 21-11_51_30](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/assets/92363071/c77f0075-3cb8-42a7-a7b2-e1ed71602f1e)

but I have the video:
![screenshot-deploy-preview-2089--boosted netlify app-2023 06 21-11_51_47](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/assets/92363071/27fcc7d4-9dd0-40a1-be57-c9c5359db6a0)

In fact I can see the warning for 1s, and then is gone and I have the video, weird isn't it?

Reply:
I can't reproduce this, even using Chrome. Is there any step or information that I might need to solve this ?

Reply:
Seems to a temporary problem, not able to reproduce anymore.